vue/sbc-common-components/src/components/SbcFeeSummary.vue

```diff
@@ -128,6 +128,11 @@ header {
   background: $BCgovBlue5;
 }
 
+ul {
+    padding: 0;
+    list-style-type: none;
+}
```

Comment:
Please advise all teams that use this library that they should update their import -- check with the respective team leads or just create new tickets for them.

Reply:
(I prefer option 2.)